### PR TITLE
greetings action

### LINKED ISSUE #163: Greeting.yml in github workflow
### Description

Please this bot and also enhance it.Considering the cases

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Tushar98644**
fixes #163

### CODE REVIEW

**Review by anmode — CHANGES_REQUESTED** [review]
We should also take into account other scenarios, such as when a mentor or collaborator creates an issue. Additionally, if a contributor who has previously created issues submits a new one, they should receive a different kind of greeting compared to a first-time contributor.

Collaborator differenly and owner differently!

**Comment by Tushar98644** [comment]
> We should also take into account other scenarios, such as when a mentor or collaborator creates an issue. Additionally, if a contributor who has previously created issues submits a new one, they should receive a different kind of greeting compared to a first-time contributor.
> 
> Collaborator differenly and owner differently!

updated , let me know if need further changes

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -1,85 +1,64 @@
...
-          fi
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          echo "::set-output name=greeting::$(node -e '
+          // Greetings mapping
+          const greetings = {
+            collaborator: "Hello, collaborator! Thank you for your contribution. We appreciate your help!",
+            owner: "Welcome, owner! We're glad to have you on board. Your expertise is valuable to us!",
+            firstTimeContributor: "Hello, first-time contributor! Thank you for opening your first issue or pull request. We're excited to collaborate with you!",
+            returningContributor: "Welcome back, returning contributor! Your continued contributions are greatly appreciated!",
```
Comment (anmode) [inline .github/workflows/greetings.yml:34]: I think there is scope for improvement in the message. See i Don't think they are ok. and also greeting way would be different....We should write hello firstTimeContributor... instead the name but we have to greet as they comes for the first. I hope you get the point

**Review by anmode — CHANGES_REQUESTED** [review]
Looks cool!

**Comment by anmode** [comment]
@Tushar98644 ?

**Comment by Tushar98644** [comment]
> @Tushar98644 ?

will update by tonight

**Comment by anmode** [comment]
@Tushar98644 ?

**Comment by Tushar98644** [comment]
> @Tushar98644 ?

sorry i was stuck on some other things , i am resumed working on it .....expect an update by midnight

**Comment by anmode** [comment]
hey? I think I should close it now.

**Pull request closed without merging**